js/storage-util.js

```diff
@@ -1,19 +1,24 @@
-/* global loadScript tryJSONparse */
+/* global loadScript tryJSONparse promisifyChrome */
 /* exported chromeLocal chromeSync */
 'use strict';
 
+promisifyChrome({
+  'storage.local': ['get', 'remove', 'set'],
+  'storage.sync': ['get', 'remove', 'set'],
+});
+
 const [chromeLocal, chromeSync] = (() => {
   return [
     createWrapper('local'),
     createWrapper('sync'),
   ];
 
   function createWrapper(name) {
-    const storage = chrome.storage[name];
+    const storage = browser.storage[name];
     const wrapper = {
-      get: data => new Promise(resolve => storage.get(data, resolve)),
-      set: data => new Promise(resolve => storage.set(data, () => resolve(data))),
-      remove: data => new Promise(resolve => storage.remove(data, resolve)),
+      get: storage.get.bind(storage),
```

Comment:
Can we drop `storage-util.js`? Now it is just a wrapper + LZString. I think we can use `LZString` directly to encode/decode the data in the editor (and maybe move it into the worker?).

Reply:
The primary purpose of chromeLocal/Sync was getValue and setValue which are used throughout the code. This can be changed in a separate PR if you'd like as it's not directly related to promisifying. As for moving into a worker, it might make sense indeed, which is also a separate PR.

Reply:
Er, to clarify, I can change chromeLocal/Sync here but only if you don't mind me changing those other places in this PR... I just wanted it to be focused on one thing.